src/olympia/git/tests/test_commands.py

```diff
@@ -170,13 +172,14 @@ def test_extract_addon_with_error_during_extraction(self):
         Path(f'{repo.git_repository_path}/.git/refs/heads/listed').touch()
         entry = GitExtractionEntry.objects.create(addon=addon)
 
-        try:
+        with pytest.raises(BrokenRefError):
```

Comment:
`self.assertRaises` is the preferred solution.  We'd only use `pytest.raises` outside of a `TestCase`.

Reply:
Why is that the preferred solution? Is that becasue `assertRaises` comes from `unittest`?

I have no problem with changing `pytest.raises` to `self.assertRaises` but we should do that everywhere, no?

- https://github.com/mozilla/addons-server/blob/855464d3890d858edf9b9a6cf09dec14c7f8666a/src/olympia/accounts/tests/test_verify.py#L33
- https://github.com/mozilla/addons-server/blob/36e03763b6e90c42dff2d8144cf173c478a08a40/src/olympia/files/tests/test_utils.py#L123

Reply:
I can't find the source code right now but iirc last time I looked `assertRaises` had some extra support for tracebacks or something.  Regardless, it's the norm in the addons-server codebase and imo it's best to pick a flavour and try to stick to it - I'm not sure it's worth specifically going through and replacing all the instances but if we were changing the code it's best to change it (that's why I suggested it 😉 at the time)